### DISCUSSION TITLE
The app is not working

### DISCUSSION THREAD

**Discussion by AzizEmir** [dynobo/normcap#515] [Feedback]
./NormCap-0.4.4-x86_64.AppImage
22:51:34 - ERROR   - normcap:211 - [QT] qtinfomsg - could not load the qt platform plugin "xcb" in "" even though it was found.
22:51:34 - ERROR   - normcap:216 - Please check if installing additional dependencies might help, see: https://dynobo.github.io/normcap/#faqs-couldnt-load-platform-plugin
22:51:34 - ERROR   - normcap:220 - If that doesn't solve it, please open an issue: https://github.com/dynobo/normcap/issues
22:51:34 - ERROR   - normcap:211 - [QT] qtfatalmsg - this application failed to start because no qt platform plugin could be initialized. reinstalling the application may fix this problem.

available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vkkhrdisplay, vnc, wayland-egl, wayland, xcb.

/tmp/.mount_NormCaK3oUSW/AppRun: satır 11: 20704 İptal edildi           "${APPDIR}/usr/python/bin/python3" -u -s -X utf8 -c "import runpy, sys; sys.path.pop(0); runpy.run_module('${BRIEFCASE_MAIN_MODULE}', run_name='__main__', alter_sys=True)" "$@"

**Comment by dynobo** [comment]
Hi @AzizEmir , thanks for reporting this issue.

I'd need further information to help you:

1. Did you read [the link mentioned in the output](https://dynobo.github.io/normcap/#faqs-couldnt-load-platform-plugin)? Did you follow those instructions?
2. Can you please provide additional info about your system:
    - Name of Linux distro & version? 
    - Are you using Wayland or Xorg compositor?

**Reply by AzizEmir** [reply]
Hello, the issue was resolved when I followed the link and installed the necessary dependencies. Thank you.
